tutorials/tensordictmodule.ipynb

```diff
@@ -0,0 +1,2042 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2b194ee2",
+   "metadata": {},
+   "source": [
+    "# The TensorDictModule"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc268c66",
+   "metadata": {},
+   "source": [
+    "Make sure to first read the tensordict tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b16eeedd",
+   "metadata": {},
+   "source": [
+    "How do we use the TensorDict it in pratice? We introduce the TensorDictModule. The TensorDictModule is an nn.Module that takes a TensorDict in his forward method. The user defines the keys that the module will take as an input and write the output in the same TensorDict at a given set of key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b2e43fd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchrl.modules import TensorDictModule, TensorDictSequence\n",
```

Comment:
I think we need to tell about `TensorDictSequence` before jumping to building stuff with it